homeassistant/components/deconz/light.py

```diff
@@ -268,6 +268,8 @@ def color_temp_kelvin(self) -> int | None:
         """Return the CT color value."""
         if self._device.color_temp is None:
             return None
+        if self._device.color_temp == 0:
+            return None
```

Comment:
What is the advantage of not combining with the above?
```python
        if self._device.color_temp is None or self._device.color_temp == 0:
            return None
```
or even
```python
        if not self._device.color_temp:
            return None
```

Reply:
I want to remember what checks there are as I have plans to rewrite things later and this is more explicit, I can put it on one line